operators/Dockerfile

```diff
@@ -15,7 +15,8 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -tags="$GO_TAGS" -a -o proces
 
 # Copy the controller-manager into a thin image
 FROM ubuntu:latest
-WORKDIR /root/
+RUN mkdir /eck && chmod 755 /eck
+WORKDIR /eck/
```

Comment:
don't we also need to run next commands as a non-root user?

Reply:
I don't think, as long as the `process-manager` file can be read by any user the Pod will be able to start.

Reply:
There is no obligation but it's a good practice to avoid to use `root`. So I'm in favor of creating an `elastic` user and using it.

Reply:
We can do that. Note that it's a good practice for a containers but not really for Kubernetes where you should use a random user id. It was one of my first mistake in production, hundreds of processes running with the same id and a pain to fix the situation when some of them are using some volumes...